ci/debian/centreon-ha-web.install

```diff
@@ -1,5 +1,5 @@
 etc/centreon-gorgone/config.d/cron.d/*      etc/centreon-gorgone/config.d/cron.d
 etc/centreon-ha/centreon_central_sync.pm    etc/centreon-ha
 etc/sudoers.d/centreon-cluster              etc/sudoers.d
-etc/sysconfig/*                             etc/sysconfig
+etc/default/*                               etc/default
```

Comment:
You will have to change the path also in the call of the files in the daemon
https://github.com/centreon/centreon-ha/blob/develop/systemd/cbd-rrd.service#L23

Reply:
It's changed

Reply:
you must be make a `override_dh_auto_build:` in debian rules, like:
```
sed -i 's#etc/sysconfig#etc/defaul#g' systemd/*.service
```